hls-graph/src/Development/IDE/Graph/Internal/Database.hs

```diff
@@ -165,7 +170,7 @@ compute db@Database{..} key mode result = do
         deps | not(null deps)
             && runChanged /= ChangedNothing
                     -> do
-            void $ forkIO $
```

Comment:
Removed this `forkIO` because it made testing reverse deps impossible, and I can't justify the need for it anyway